subsys/dfu/src/dfu_target_modem.c

```diff
@@ -153,9 +155,10 @@ int dfu_target_modem_init(void)
 	return 0;
 }
 
-int dfu_target_modem_offset(void)
+int dfu_target_modem_offset(size_t *out)
```

Comment:
This does not match the signature in the declaration in the .h file

Reply:
See answer/question on the other comment.